OpenEdXMobile/src/main/java/org/edx/mobile/module/prefs/AppInfoPrefs.kt

```diff
@@ -0,0 +1,68 @@
+package org.edx.mobile.module.prefs
+
+import android.content.Context
+import dagger.hilt.android.qualifiers.ApplicationContext
+import org.edx.mobile.view.BulkDownloadFragment
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class AppInfoPrefs @Inject constructor(@ApplicationContext context: Context) :
+    PrefBaseManager(context, Pref.APP_INFO) {
+    init {
+        migrateData(PrefBaseManager(context, Pref.COURSE_CALENDAR_PREF))
+        migrateData(PrefBaseManager(context, Pref.VIDEOS))
+    }
+
+    var appVersionCode: Long
+        get() = getLong(APP_VERSION_CODE)
+        set(code) = put(APP_VERSION_CODE, code)
+
+    var appVersionName: String
+        get() = getString(APP_VERSION_NAME, "0.0.0")
+        set(code) = put(APP_VERSION_NAME, code)
+
+    var appRating: Float
+        get() = getFloat(APP_RATING)
+        set(appRating) = put(APP_RATING, appRating)
+
+    var lastRatedVersion: String
+        get() = getString(LAST_RATED_VERSION, "0.0.0")
+        set(versionName) = put(LAST_RATED_VERSION, versionName)
+
+    var whatsNewShownVersion: String?
+        get() = getString(WHATS_NEW_SHOWN_FOR_VERSION)
+        set(version) = put(WHATS_NEW_SHOWN_FOR_VERSION, version)
+
+    fun setSyncAlertPopupDisabled(courseName: String) {
```

Comment:
We should rename the function to `disableCalendarSyncPopupForCourse` because it's no longer under Calendar Prefs.

Reply:
The new name `setCalendarSyncAlertPopupDisabled` isn't making much sense. :/